packages/common/src/types/job.ts

```diff
@@ -1,19 +1,23 @@
-export const enum JobStatusType {
+export enum JobType {
+    COMPILE_PROJECT = 'COMPILE_PROJECT',
```

Comment:
what's the advantage of making this a job specific property instead of using steps ?

Reply:
This is to differentiate the `jobResults` you get from each job. So `COMPILE_PROJECT` jobs will have a `{ projectUuid }` available on successfully completing